exotic/api/nea.py

```diff
@@ -307,9 +307,7 @@ def _get_params(self, data):
                 rprserr = ((rperr / rs) ** 2 + (-rp * rserr / rs ** 2) ** 2) ** 0.5
                 rprs = rp / rs
 
-        if data['pl_ratdor'] is None:
-            data['pl_ratdor'] = pow((data['pl_orbper'] / 365) ** 2, 1 / 3) / (data['st_rad'] * R_SUN.to('au')).value
-        elif data['pl_ratdor'] < 1 or np.isnan(data['pl_ratdor']):
+        if data['pl_ratdor'] is None or data['pl_ratdor'] < 1 or np.isnan(data['pl_ratdor']):
```

Comment:
```
if data['pl_ratdor'] is None or np.isnan(data['pl_ratdor']) or data['pl_ratdor'] < 1.:
    data['pl_ratdor'] = pow((data['pl_orbper'] / 365.) ** 2, 1. / 3.) / (data['st_rad'] * R_SUN.to('au')).value
```
- Check if none, then if `isnan`, then since it passes those the number can actually be compared.
- Added decimal values in reassignment to guarantee float calculations are respected and not inadvertently rounded.

Reply:
I'd be a little careful with the chain of events here.. If `data['pl_ratdor']` is indeed None, which can happen (since NEA doesn't have a complete set of parameters for every target), python will throw an error if you try to compare None to being `<1`, which is why I arranged that condition to be first in this sequence. Otherwise, you can't string it all together and combine it with `or`

Reply:
@pearsonkyle - given this, should we just close this PR and not pull it into dev?

Reply:
I updated the PR as requested by @jpl-jengelke. Even if `data['pl_ratdor']` is None, it will never be compared to 1, since `or` is a short-circuit operator. Even if it did, it would not raise an exception, since `None < 1 == True` in Python.